### ISSUE TITLE
As a thrid party I want to be able to add contents through a google spreasheet to opendatahub, to simplify insertion of new data

### ISSUE THREAD

**Issue by bertolla** [noi-techpark/bdp-commons#114]

**Comment by bertolla** [comment]
Inserted data can already be seen here https://geodata.integreen-life.bz.it/edi/wms?service=WMS&version=1.1.0&request=GetMap&layers=edi%3Acreative%20points%20on%20map&bbox=11.1433265%2C46.3395693%2C11.9563958%2C46.8330732&width=768&height=466&srs=EPSG%3A4326&format=application/openlayers

**Comment by bertolla** [comment]
and can already be tested with `Tschiwiti` http://tomcat.testingmachine.eu/reader2/api/v2/flat/CreativeIndustry

**Comment by bertolla** [comment]
It is now ready for testing. You shoult all have a document that I shared called CreativeIndustries(https://docs.google.com/spreadsheets/d/1I5Zj7JHprwLhzl9ktXJO-7v7x3rh1ysGbiblZgZYCXU/edit#gid=1952520682). I wrote a rough user guide in the README of the project https://github.com/noi-techpark/bdp-commons/tree/development/data-collectors/spreadsheets-google. Try it out pls!

**Comment by LucaMiotto** [comment]
- Information from the spreadsheet are shown correctly on the map and in the description box for each point.
- I've tried to insert a new location with address "Via Alessandro Volta 13 39100 Bolzano". First time didn't work so I used OSM Nomination, as readme says, but the naming seams to be correct. I've also tried the german version "Alessandro-Volta-Straße 13 39100 Bozen" but probably didn't synced anymore.
- Would be useful find a way to let the "inserter" when synchronization happens. Changing color of the cell or something similar.
